src/addons/Portal/Portal.js

```diff
@@ -363,6 +363,14 @@ class Portal extends Component {
     )
 
     this.portal = this.node.firstElementChild
+    // don't take focus away from portals that close on blur
+    if (!closeOnTriggerBlur) {
+      this.triggerNode = document.activeElement
+      this.portal.setAttribute('tabindex', '-1')
+      this.portal.style.outline = 'none'
+      // wait a tick for things like popups which need to calculate where the popup shows up
+      setTimeout(() => this.portal && this.portal.focus())
```

Comment:
I'd like to pull this side effect out of the render method if at all possible.  I haven't the bandwidth at the moment to find a better place myself, but open to suggestions.